### ISSUE TITLE
the val mAP is lower using multi gpu?

### ISSUE THREAD

**Issue by ghoshaw** [RangiLyu/nanodet#392]
HI, I use the nano-plus train 2D detect, and I found the val mAp is higher using 1 Gpu, if it means I should adjust the lr when using multi gpu? the batchsize_per_gpu is the same.
1 gpu
![image](https://user-images.githubusercontent.com/23494873/150708995-ca6d410c-b1c7-4bcf-8067-dcaf3f28d6de.png)

8 gpus
![image](https://user-images.githubusercontent.com/23494873/150709201-e562c696-9d77-4705-99a9-10823f5e4eec.png)

**Comment by RangiLyu** [comment]
Yes, the total batchsize is changed so you need to adjust the learning rate.

**Comment by ghoshaw** [comment]
ok, thanks, I will give it a try.
